sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -680,4 +681,22 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
     sessionState.close()
     SparkSQLEnv.stop()
   }
+
+  test("SPARK-39068: support in-memory catalog and running concurrently") {
+    val extraConf = Seq("-c", "spark.sql.catalogImplementation=in-memory")
```

Comment:
`spark.sql.catalogImplementation=in-memory` -> `${StaticSQLConf.CATALOG_IMPLEMENTATION.key}=in-memory`?

Reply:
thanks updated